lib/perl/Genome/Model/Tools/Somatic/FilterFalsePositives.pm

```diff
@@ -636,6 +665,10 @@ sub fails_homopolymer_check {
 sub readcount_program {
     my $self = shift;
     my $reference = $self->reference;
+    if($self->bam_readcount_version) {
+      my $bam_rc = "bam-readcount" . $self->bam_readcount_version;
```

Comment:
it seemed like getting the bam-readcount path from that module could be a little complicated since `sub readcount_path` uses `$self->use_version` instead of a parameter to that routine.